test/fixtures/simple/index.js

```diff
@@ -1,8 +1,8 @@
-import foo from './foo.css'
```

Comment:
These files don't use css modules. So previously when testing in "inject" mode "foo" was the css content, and when it "extract" mode "foo" was undefined.

Now in extract mode there is no default export so this throws an error, so I've remove the mention of it